alpheios_nemo_ui/data/assets/images/alpheios-logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" width="69.8px" height="50.9px" viewBox="0 0 69.8 50.9" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It's probably better to remove `width` and `height` and set them with external CSS rules (i.e. in a CSS/Sass file). With it, we'll be able to use the same SVG file in different locations, in various sizes.

Reply:
Yes, it is set inside css rules.
But first it loads without any css - and it looks like the following on a screeb:
   - first this logo stretches for all the screen
   - then it gets size from css rules

After I have reloaded the page a dozen times - my eyes were tied with these jumps :)
this width and height have no css priority at all - so it couldn't make any problems for css - but this decreases "image jumps on load"

Reply:
Oh, that makes sense! Totally agree with your approach then. 👍

Reply:
> Yes, it is set inside css rules.
> But first it loads without any css - and it looks like the following on a screen:
> 
> * first this logo stretches for all the screen
> * then it gets size from css rules
> 
> After I have reloaded the page a dozen times - my eyes were tied with these jumps :)
> this width and height have no css priority at all - so it couldn't make any problems for css - but this decreases "image jumps on load"

oh! Is this the cause for #89  do you think?

Reply:
Yes, I read that issue after all my PRs and I think that it is the way css files are uploaded via flask,
first all HTML layout is loaded and only then css/jss, and css are applied after it is loaded completely.

Reply:
ok, thanks. I will note this in the other issue.